docs/monoDrive_home/Simulator-Configuration.md

```diff
@@ -2,135 +2,132 @@
  
 ## Simulator Configuration
 
-```
+``` json
 {
-   "server_ip":"127.0.0.1",
-   "map":"Almono",
-   "server_port":8999,
-   "simulation_mode":2,
-   "ego_config": {	   
-		"body": {	     
-			"type": "/Game/Vehicles/crossover_monoDrive_01.crossover_monoDrive_01_C",
-			"color": "Carpaint_White.Carpaint_White"
-		}
-	},
-   "traffic_configuration":{
-      "max_vehicles":20,
-      "min_desired_speed":-20.0,
-      "max_desired_speed":10.0,
-      "swarm_traffic":true,
-      "spawn_leash_distance":15000,
-      "min_spawn_range":2000,
-      "vertical_offset":25,
-      "overtake_slower_vehicles":true
-   },
-   "phys_materials":{
-      "Aluminum":{
-         "specular_exponent":15.0,
-         "specular_coefficient":0.95,
-         "diffuse_coefficient":0.26,
-         "dielectric_constant":10.0,
-         "roughness":0.15
+   "client_ip": "127.0.0.1",
+   "client_settings": {
+      "gui": {
+         "fps": 1.0
       },
-      "Asphalt":{
-         "specular_exponent":1.0,
-         "specular_coefficient":0.03,
-         "diffuse_coefficient":0.65,
-         "dielectric_constant":7.0,
-         "roughness":0.65
+      "logger": {
+         "control": "info",
+         "network": "info",
+         "scenario": "info",
+         "sensor": "debug",
+         "simulator": "info",
+         "vehicle": "info"
       },
-      "Concrete":{
-         "specular_exponent":0.0,
-         "specular_coefficient":0.1,
-         "diffuse_coefficient":0.65,
-         "dielectric_constant":7.0,
-         "roughness":0.0
+      "map": {
+         "gis_anchor": {
+               "x": 0,
+               "y": 0,
+               "z": 0
+         },
+         "point_delta": 100.0
+      }
+   },
+   "ego_config": {
+      "body": {
+         "color": "CarPaint_White.CarPaint_White",
+         "type": "Blueprint'/Game/Vehicles/crossover_monoDrive_01.crossover_monoDrive_01_C'"
       },
-      "Glass":{
-         "specular_exponent":80.0,
-         "specular_coefficient":0.80,
-         "diffuse_coefficient":0.15,
-         "dielectric_constant":4.0,
-         "roughness":0.10
+      "vehicle_dynamics": "physx"
+   },
+   "id": "simulator_test",
+   "map": "Almono",
+   "phys_materials": {
```

Comment:
It looks like the default physical materials in the C++ client are not the latest values. Replace these guys with the ones from the LV client. Opened [this issue](https://github.com/monoDriveIO/monodrive-client/issues/44) to address the problem. Below should be the latest:

```json
{
    "phys_materials": {
    "Aluminum": {
      "specular_exponent": 15.0,
      "specular_coefficient": 0.95,
      "diffuse_coefficient": 0.26,
      "dielectric_constant": 10.0,
      "roughness": 0.15
    },
    "Asphalt": {
      "specular_exponent": 1.0,
      "specular_coefficient": 0.03,
      "diffuse_coefficient": 0.65,
      "dielectric_constant": 7.0,
      "roughness": 0.65
    },
    "Concrete": {
      "specular_exponent": 0.0,
      "specular_coefficient": 0.1,
      "diffuse_coefficient": 0.65,
      "dielectric_constant": 7.0,
      "roughness": 0.0
    },
    "Glass": {
        "specular_exponent": 80.0,
        "specular_coefficient": 0.80,
        "diffuse_coefficient": 0.15,
        "dielectric_constant": 4.0,
        "roughness": 0.10
      },
    "Gravel": {
        "specular_exponent": 10.0,
        "specular_coefficient": 0.1,
        "diffuse_coefficient": 0.95,
        "dielectric_constant": 7.0,
        "roughness": 0.48
      },
    "LandScape": {
        "specular_exponent": 10.0,
        "specular_coefficient": 0.50,
        "diffuse_coefficient": 0.50,
        "dielectric_constant": 10.0,
        "roughness": 0.60
      },
    "Plastic": {
        "specular_exponent": 25.0,
        "specular_coefficient": 0.40,
        "diffuse_coefficient": 0.60,
        "dielectric_constant": 2.8,
        "roughness": 0.09
      },
    "ReflectiveMaterial": {
      "specular_exponent": 10.0,
      "specular_coefficient": 0.5,
      "diffuse_coefficient": 0.5,
      "dielectric_constant": 1.0,
      "roughness": 1.0
    },
    "Road": {
      "specular_exponent": 1.0,
      "specular_coefficient": 0.1,
      "diffuse_coefficient": 0.65,
      "dielectric_constant": 7.0,
      "roughness": 0.65
    },
    "RoadPaint": {
        "specular_exponent": 10.0,
        "specular_coefficient": 10.0,
        "diffuse_coefficient": 2.0,
        "dielectric_constant": 7.0,
        "roughness": 1.0
      },
    "Steel": {
      "specular_exponent": 26.0,
      "specular_coefficient": 1.0,
      "diffuse_coefficient": 0.33,
      "dielectric_constant": 0.0,
      "roughness": 0.23
    },
    "Tire": {
      "specular_exponent": 25.0,
      "specular_coefficient": 0.40,
      "diffuse_coefficient": 0.60,
      "dielectric_constant": 2.8,
      "roughness": 0.15
    },
    "Tree": {
      "specular_exponent": 10.0,
      "specular_coefficient": 0.05,
      "diffuse_coefficient": 0.97,
      "dielectric_constant": 10.0,
      "roughness": 0.60
    },
    "Wheel": {
      "specular_exponent": 92.0,
      "specular_coefficient": 0.95,
      "diffuse_coefficient": 0.06,
      "dielectric_constant": 10.0,
      "roughness": 0.15
    }
  }
```